Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/BoundsTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class BoundsTests : TypeTester<Bounds>
+    {
+        public static (Bounds deserialized, string serialized)[] representations = new[] {
+            (new Bounds(new Vector3(1, 2, 3), new Vector3(4, 5, 6)), @"{""xMin"":1,""yMin"":2,""zMin"":3,""sizeX"":4,""sizeY"":5,""sizeZ"":6}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5190738)